### PR TITLE
Add company 8fit GmbH

### PR DESCRIPTION
**Pull request by osartun**
We currently don't have any job openings for Django developers right now which is why I used a link to an old facebook post as proof. Hope that works?
Either way: We're located in Berlin and we're using Django. So, that should meet the list's criteria I think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Something is missing?  Create a pull request!
         [proof](https://news.ycombinator.com/item?id=8822940)
         ,
         [github](https://github.com/42reports)
+    * [8fit GmbH](https://8fit.com) –
+        [proof](https://www.facebook.com/berlinstartupjobs/posts/1606458446068530?__xts__[0]=68.ARDBL2KBa2FnNqDOzg4kD5Jkz1pUHgJxePJfrR0ez323STOvhZtaR2FRi_K28dZMLv9A6eW2KhS4LweCvbHNsyYWB3xwlhsxcGbhEfy9yG1SlH3hhoem1CIN7jumYdKspd7eyTGJ-pEeBAKHL8io-1PDl0uypV3QVNxrC2BHXEfnj69kqIQWRwTUb251BK2RLPmyeU6xe-L9NUTGrIgljsVQkYFc0z5aZpgp2AkX1mcTXK9BM6pJh0Z-RF7P-x4ed73lUnExYaloQZHXZOiWU-w6vUzew73iy5H1yPSZ6lz5Gk6v_YFWyvvztBjvn4hmLZamocm9FjlSg3nB8Ew5MCXhqw&__tn__=-R)
```
Comment (hartwork) [inline README.md]: Hi Oliver,

cool you found your way here! The pull request looks excellent except for one bit: Can you crop the URL down to https://www.facebook.com/berlinstartupjobs/posts/1606458446068530? That seems to still work but is less bloaty (and makes the string a well-formed URI (which something with square brackets is not as that's a "reserved" character) with regard to [RFC 3986](https://www.ietf.org/rfc/rfc3986.txt), technically).

Thanks!

**Review by hartwork — CHANGES_REQUESTED** [review]
:+1: Please check the one comment above.

File: README.md (hunk 1 above)
Comment (osartun) [reply README.md]: Done! :)
Good point, didn't try out removing all the excess.

**Review by hartwork — CHANGES_REQUESTED** [review]
Thanks for the URL fix!

You'll probably start laughing at me but there's one last thing I'd ask to fix that I only notice just now: You're not using the same emdash that all the other entries use:
```python
In [1]: ord('—')  # other posts, aka ctrl+shift+u 2 0 1 4
Out[1]: 8212

In [2]: ord('–')  # yours
Out[2]: 8211

In [3]: ord('-')  # minus
Out[3]: 45

In [4]: from html.entities import name2codepoint

In [5]: name2codepoint['mdash']  # see first case
Out[5]: 8212
```

If you fix that thing, I don't need to fix it and you passed the most picky review on the planet (maybe) :smile: 

What do you think?

**Comment by osartun** [comment]
@hartwork The one thing that I didn't copy & paste before 😄 
https://github.com/hartwork/django-berlin/pull/1/commits/edc7d351a5c10438f9366b380da1633b24b56d66

**Comment by hartwork** [comment]
Thank you!

**Pull request merged**